src/importexport/musicxml/internal/musicxml/import/importmusicxmlpass2.cpp

```diff
@@ -8149,6 +8162,11 @@ void MusicXmlParserNotations::technical()
             m_notations.push_back(notation);
         } else if (m_e.name() == "harmonic") {
             harmonic();
+        } else if (m_e.name() == "handbell") {
+            const std::vector<XmlStreamReader::Attribute> attributes = m_e.attributes();
+            convertArticulationToSymId(m_e.readText(), id);
+            m_notations.push_back(Notation::notationWithAttributes(String::fromAscii(m_e.name().ascii()),
+                                                                   attributes, u"technical", id));
```

Comment:
Why not `m_e.attributes()` here too?

Reply:
After `readText` the attributes are no longer accessible.

Reply:
Why then does it work above?

Reply:
Hmm, I think I see now...